TESTS.md

```diff
@@ -292,6 +292,28 @@ test
   ,)
 ```
 
+# Record syntax
+
+Pattern matching, short
+
+```haskell
+fun Rec {alpha = beta, gamma = delta, epsilon = zeta, eta = theta, iota = kappa} = do
+  beta + delta + zeta + theta + kappa
```

Comment:
I think code block braces are very different syntactic case. I'd prefer to exclude them from this discussion.

As for record declarations:
1. (My personal opinion) Record declaration must be consistent with record access/update/etc syntax, and latter must be consistent with list and tuple syntax.
2. (My personal opinion) Record declaration must be consistent with type tuple.
```haskell
data User = User (Id, String)
newtype Bourne a = Bourne (Identity, a)
```

I'd love to hear @chrisdone's and @tibbe's thoughts.